tests/utils/0010_mkfs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DEV=${FS_PARAMS:21};
```

Comment:
Hackish. This assumes a lot of the exact fs-uri parameters, so it will easily break. Introduce a new parameter in stead that needs to be set by the user (ZDEV or something like that) running the tests.

sudo FS_PARAMS="--fs-uri=zenfs://dev:nvme0n1" ZDEV=nvme0n1 ./run.sh util-tests utils

Reply:
Ok, ZDEV looks clean.

Yes, you are right, readrandom is more of a sanity test.
So, I have added a check to do a verify after a back up and verify after a restore using "ldb dump" similar to the scripts you pointed out, the steps are as below:
After back up:
Do ldb dump of the ondisk database in to a dumpfileA
Do ldb dump of the backed up directory into dumpfileB
Compare dumpfile A and B, error if it does not match.

After restore:
Do ldb dump of the ondisk database(newly restored) in to a dumpfileA
Do ldb dump of the source directory (from where restore was done) into dumpfileB
Compare dumpfile A and B, error if it does not match.